src/main/content/_assets/css/guide.scss

```diff
@@ -43,26 +43,25 @@ header {
 }
 
 #guide-content h1 {
-    font-size: 28px;
+    font-size: 2em;
```

Comment:
is there a reason to use em here instead of rem?

Reply:
whoops, fixed now 😆